### ISSUE TITLE
Re Open Exception: System.InvalidOperationException: The gRPC channel URI 'http://:' could not be parsed. since it's not fixed

### ISSUE THREAD

**Issue by milen-denev** [Azure/azure-functions-dotnet-worker#2248]
### Description

```
Result: Function 'ONE_OF_18_FUNCTION_SERVICES', Invocation id 'a1ec01be-3a41-4f69-b11f-e0e0ba5a2f76': An exception was thrown by the invocation.
Exception: System.InvalidOperationException: The gRPC channel URI 'http://:' could not be parsed.
   at Microsoft.Azure.Functions.Worker.Extensions.Rpc.ConfigurationExtensions.GetFunctionsHostGrpcUri(IConfiguration configuration) in D:\a\_work\1\s\extensions\Worker.Extensions.Rpc\src\ConfigurationExtensions.cs:line 35
   at Microsoft.Azure.Functions.Worker.GrpcHttpClientBuilderExtensions.<>c.<ConfigureForFunctionsHostGrpc>b__0_0(GrpcClientFactoryOptions options, IConfiguration config) in D:\a\_work\1\s\extensions\Worker.Extensions.Rpc\src\GrpcHttpClientBuilderExtensions.cs:line 34
   at Microsoft.Extensions.Options.OptionsFactory`1.Create(String name)
   at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)
   at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)
   at System.Lazy`1.CreateValue()
   at Microsoft.Extensions.Options.OptionsCache`1.GetOrAdd[TArg](String name, Func`3 createOptions, TArg factoryArgument)
   at Grpc.Net.ClientFactory.Internal.GrpcCallInvokerFactory.CreateInvoker(EntryKey key)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Grpc.Net.ClientFactory.Internal.GrpcCallInvokerFactory.CreateInvoker(String name, Type type)
   at Grpc.Net.ClientFactory.Internal.DefaultGrpcClientFactory.CreateClient[TClient](String name)
   at Microsoft.Extensions.DependencyInjection.GrpcClientServiceExtensions.<>c__DisplayClass7_0`1.<AddGrpcHttpClient>b__2(IServiceProvider s)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitFactory(FactoryCallSite factoryCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitIEnumerable(IEnumerableCallSite enumerableCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitRootCache(ServiceCallSite callSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.CreateServiceAccessor(ServiceIdentifier serviceIdentifier)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at Microsoft.Azure.Functions.Worker.ServiceBusExtensionStartup.<>c.<Configure>b__0_0(IServiceProvider sp) in D:\a\_work\1\s\extensions\Worker.Extensions.ServiceBus\src\ServiceBusExtensionStartup.cs:line 21
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitFactory(FactoryCallSite factoryCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.DynamicServiceProviderEngine.<>c__DisplayClass2_0.<RealizeService>b__0(ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.Azure.Functions.Worker.Converters.DefaultInputConverterProvider.GetOrCreateConverterInstance(Type converterType) in D:\a\_work\1\s\src\DotNetWorker.Core\Converters\Converter\DefaultInputConverterProvider.cs:line 74
   at Microsoft.Azure.Functions.Worker.Converters.DefaultInputConverterProvider.<GetOrCreateConverterInstance>b__8_0(String key, String converterTypeAssemblyQualifiedName) in D:\a\_work\1\s\src\DotNetWorker.Core\Converters\Converter\DefaultInputConverterProvider.cs:line 101
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd[TArg](TKey key, Func`3 valueFactory, TArg factoryArgument)
   at Microsoft.Azure.Functions.Worker.Converters.DefaultInputConverterProvider.GetOrCreateConverterInstance(String converterTypeName) in D:\a\_work\1\s\src\DotNetWorker.Core\Converters\Converter\DefaultInputConverterProvider.cs:line 91
   at Microsoft.Azure.Functions.Worker.Context.Features.DefaultInputConversionFeature.GetConverterFromContext(ConverterContext context) in D:\a\_work\1\s\src\DotNetWorker.Core\Context\Features\DefaultInputConversionFeature.cs:line 131
   at Microsoft.Azure.Functions.Worker.Context.Features.DefaultInputConversionFeature.ConvertAsync(ConverterContext converterContext) in D:\a\_work\1\s\src\DotNetWorker.Core\Context\Features\DefaultInputConversionFeature.cs:line 39
   at Microsoft.Azure.Functions.Worker.Context.Features.DefaultFunctionInputBindingFeature.ConvertAsync(FunctionContext context, FunctionParameter parameter, IConverterContextFactory converterContextFactory, IInputConversionFeature inputConversionFeature, Object source) in D:\a\_work\1\s\src\DotNetWorker.Core\Context\Features\DefaultFunctionInputBindingFeature.cs:line 157
   at Microsoft.Azure.Functions.Worker.Context.Features.DefaultFunctionInputBindingFeature.BindFunctionInputAsync(FunctionContext context) in D:\a\_work\1\s\src\DotNetWorker.Core\Context\Features\DefaultFunctionInputBindingFeature.cs:line 55
   at DenevCloud.Core.Azure.FunctionsV2.DirectFunctionExecutor.ExecuteAsync(FunctionContext context) in Q:\Projects\Pro\Denev Cloud\DenevCloud\DenevCloud.Core.Azure.FunctionsV2\Microsoft.Azure.Functions.Worker.Sdk.Generators\Microsoft.Azure.Functions.Worker.Sdk.Generators.FunctionExecutorGenerator\GeneratedFunctionExecutor.g.cs:line 48
   at Microsoft.Azure.Functions.Worker.OutputBindings.OutputBindingsMiddleware.Invoke(FunctionContext context, FunctionExecutionDelegate next) in D:\a\_work\1\s\src\DotNetWorker.Core\OutputBindings\OutputBindingsMiddleware.cs:line 13
   at Microsoft.Azure.Functions.Worker.FunctionsApplication.InvokeFunctionAsync(FunctionContext context) in D:\a\_work\1\s\src\DotNetWorker.Core\FunctionsApplication.cs:line 77
Stack:    at Microsoft.Azure.Functions.Worker.Extensions.Rpc.ConfigurationExtensions.GetFunctionsHostGrpcUri(IConfiguration configuration) in D:\a\_work\1\s\extensions\Worker.Extensions.Rpc\src\ConfigurationExtensions.cs:line 35
   at Microsoft.Azure.Functions.Worker.GrpcHttpClientBuilderExtensions.<>c.<ConfigureForFunctionsHostGrpc>b__0_0(GrpcClientFactoryOptions options, IConfiguration config) in D:\a\_work\1\s\extensions\Worker.Extensions.Rpc\src\GrpcHttpClientBuilderExtensions.cs:line 34
   at Microsoft.Extensions.Options.OptionsFactory`1.Create(String name)
   at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)
   at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)
   at System.Lazy`1.CreateValue()
   at Microsoft.Extensions.Options.OptionsCache`1.GetOrAdd[TArg](String name, Func`3 createOptions, TArg factoryArgument)
   at Grpc.Net.ClientFactory.Internal.GrpcCallInvokerFactory.CreateInvoker(EntryKey key)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Grpc.Net.ClientFactory.Internal.GrpcCallInvokerFactory.CreateInvoker(String name, Type type)
   at Grpc.Net.ClientFactory.Internal.DefaultGrpcClientFactory.CreateClient[TClient](String name)
   at Microsoft.Extensions.DependencyInjection.GrpcClientServiceExtensions.<>c__DisplayClass7_0`1.<AddGrpcHttpClient>b__2(IServiceProvider s)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitFactory(FactoryCallSite factoryCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitIEnumerable(IEnumerableCallSite enumerableCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitRootCache(ServiceCallSite callSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.CreateServiceAccessor(ServiceIdentifier serviceIdentifier)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at Microsoft.Azure.Functions.Worker.ServiceBusExtensionStartup.<>c.<Configure>b__0_0(IServiceProvider sp) in D:\a\_work\1\s\extensions\Worker.Extensions.ServiceBus\src\ServiceBusExtensionStartup.cs:line 21
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitFactory(FactoryCallSite factoryCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.DynamicServiceProviderEngine.<>c__DisplayClass2_0.<RealizeService>b__0(ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.Azure.Functions.Worker.Converters.DefaultInputConverterProvider.GetOrCreateConverterInstance(Type converterType) in D:\a\_work\1\s\src\DotNetWorker.Core\Converters\Converter\DefaultInputConverterProvider.cs:line 74
   at Microsoft.Azure.Functions.Worker.Converters.DefaultInputConverterProvider.<GetOrCreateConverterInstance>b__8_0(String key, String converterTypeAssemblyQualifiedName) in D:\a\_work\1\s\src\DotNetWorker.Core\Converters\Converter\DefaultInputConverterProvider.cs:line 101
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd[TArg](TKey key, Func`3 valueFactory, TArg factoryArgument)
   at Microsoft.Azure.Functions.Worker.Converters.DefaultInputConverterProvider.GetOrCreateConverterInstance(String converterTypeName) in D:\a\_work\1\s\src\DotNetWorker.Core\Converters\Converter\DefaultInputConverterProvider.cs:line 91
   at Microsoft.Azure.Functions.Worker.Context.Features.DefaultInputConversionFeature.GetConverterFromContext(ConverterContext context) in D:\a\_work\1\s\src\DotNetWorker.Core\Context\Features\DefaultInputConversionFeature.cs:line 131
   at Microsoft.Azure.Functions.Worker.Context.Features.DefaultInputConversionFeature.ConvertAsync(ConverterContext converterContext) in D:\a\_work\1\s\src\DotNetWorker.Core\Context\Features\DefaultInputConversionFeature.cs:line 39
   at Microsoft.Azure.Functions.Worker.Context.Features.DefaultFunctionInputBindingFeature.ConvertAsync(FunctionContext context, FunctionParameter parameter, IConverterContextFactory converterContextFactory, IInputConversionFeature inputConversionFeature, Object source) in D:\a\_work\1\s\src\DotNetWorker.Core\Context\Features\DefaultFunctionInputBindingFeature.cs:line 157
   at Microsoft.Azure.Functions.Worker.Context.Features.DefaultFunctionInputBindingFeature.BindFunctionInputAsync(FunctionContext context) in D:\a\_work\1\s\src\DotNetWorker.Core\Context\Features\DefaultFunctionInputBindingFeature.cs:line 55
   at DenevCloud.Core.Azure.FunctionsV2.DirectFunctionExecutor.ExecuteAsync(FunctionContext context) in Q:\Projects\Pro\Denev Cloud\DenevCloud\DenevCloud.Core.Azure.FunctionsV2\Microsoft.Azure.Functions.Worker.Sdk.Generators\Microsoft.Azure.Functions.Worker.Sdk.Generators.FunctionExecutorGenerator\GeneratedFunctionExecutor.g.cs:line 48
   at Microsoft.Azure.Functions.Worker.OutputBindings.OutputBindingsMiddleware.Invoke(FunctionContext context, FunctionExecutionDelegate next) in D:\a\_work\1\s\src\DotNetWorker.Core\OutputBindings\OutputBindingsMiddleware.cs:line 13
   at Microsoft.Azure.Functions.Worker.FunctionsApplication.InvokeFunctionAsync(FunctionContext context) in D:\a\_work\1\s\src\DotNetWorker.Core\FunctionsApplication.cs:line 77 
```

### Steps to reproduce

Clicking "Publish" in Visual Studio.

**Comment by milen-denev** [comment]
#2121

**Comment by milen-denev** [comment]
This is not a joke, seriously this whatever bug, caused downtime on 18 services running in Azure Functions. It is the second time some bug / issue with the worker runtime causes some serious trouble. I am very disappointed from this service's issues.

**Comment by kshyju** [comment]
@milen-denev The fix was made to functions host and will be included in the [release 4.3.0](https://github.com/Azure/azure-functions-host/releases/tag/v4.30.0). The deployment of this release has not started as of this writing. Until then, you could use the [workaround mentioned here](https://github.com/Azure/azure-functions-dotnet-worker/issues/2121#issuecomment-1847281008) to mitigate the issue.

Thank you for being patient. Don't hesitate to let us know if you run into issues with isolated model.

**Comment by TomGroeneboer** [comment]
This issue isn't solved yet until 4.30.0 is released on Azure instances. I tried forcing the framework version to 4.30.0 but I ended up with 4.29.1 still.

Any idea when that will be deployed?

**Comment by milen-denev** [comment]
I am stuck with 4.28.4.4

**Comment by milen-denev** [comment]
#2260

**Comment by quinterojose** [comment]
WEBSITE_USE_PLACEHOLDER_DOTNETISOLATED = 0 seems to have worked for us.

Currently on .NET 8 using .ServiceBus 5.16.0 and .Worker 1.21.0

Nothing special as far as config in DI.

Pulling connection from standard ServiceBusConnection setting and confirmed this setting is properly set up in app service config.

Host runtime is 4.28.4.4

**Comment by kshyju** [comment]
Thanks for the update @quinterojose A future version of host deployment will handle this in a better way and the app setting can be then removed. I will share an update when the host deployment is done. Thank you!

**Comment by jaliyaudagedara** [comment]
I can see [functions host: 4.30.0 is released.](https://github.com/Azure/azure-functions-host/releases/tag/v4.30.0)

Just created a function app in Azure, and it's still in old version:
![image](https://github.com/Azure/azure-functions-dotnet-worker/assets/5653381/f213bf62-18c7-422b-a530-1f326ac180a2)

Tried [pinning to a specific version](https://learn.microsoft.com/en-us/azure/azure-functions/set-runtime-version?tabs=portal#manual-version-updates-on-linux), 
```
az functionapp config set --name func-dotnet8-isolated-bindings `
    --resource-group rg-az-functions-isolated `
    --linux-fx-version 'DOCKER|mcr.microsoft.com/azure-functions/dotnet-isolated:4.30.0-dotnet-isolated8.0-appservice'
```
![image](https://github.com/Azure/azure-functions-dotnet-worker/assets/5653381/586d34ac-3074-49e2-b91e-fecc1603533d)

**Comment by mattchenderson** [comment]
@jaliyaudagedara The release being tagged on the repo does not mean that it has been fully deployed. It's actually the first step in the process, as it defines the payload going into that deployment. As you have noted, this is not yet available on Linux.

**Comment by jaliyaudagedara** [comment]
@mattchenderson, got it!

**Comment by milen-denev** [comment]
#2121

**Comment by PabloPaezSheridan** [comment]
> @milen-denev The error happens only when using Microsoft.Azure.Functions.Worker.Extensions.ServiceBus 5.14.0 (which [added message settlement support](https://github.com/Azure/azure-functions-dotnet-worker/releases/tag/servicebus-extension-5.14.0)) or above. Is it okay for your app to use the previous version (5.13.0) until the fix is out?
> 
> The fix was made to functions host and will be included in the [release 4.3.0](https://github.com/Azure/azure-functions-host/releases/tag/v4.30.0). The deployment of this release has not started as of this writing. Until then, you could also use the [workaround mentioned here](https://github.com/Azure/azure-functions-dotnet-worker/issues/2121#issuecomment-1847281008) to mitigate the issue, if downgrading the package version is not an option.
> 
> Thank you for being patient. Don't hesitate to let us know if you run into issues with isolated model.

If it helps this 'The gRPC channel URI 'http://:' could not be parsed.' happen to me with 5.20.1 and after a lot of effort the solution that works for me was to downgrade, as you said, to 5.13.0. thank you soo much

**Comment by mregni** [comment]
Just a short FYI, it looks like it's working for me on version 5.22.1 again.
